driver-core/src/main/com/mongodb/internal/async/AsyncBatchCursor.java

```diff
@@ -72,4 +75,22 @@ public interface AsyncBatchCursor<T> extends Closeable {
      */
     @Override
     void close();
+
+    default void exhaustCursor(final SingleResultCallback<List<List<T>>> finalCallback) {
```

Comment:
Given that `exhaustCursor` is a method on `AsyncBatchCursor`, we don't need to clarify that it applies to a cursor:

```suggestion
    default void exhaust(final SingleResultCallback<List<List<T>>> finalCallback) {
```